Libs/MRML/Logic/vtkMRMLSliceLogic.cxx

```diff
@@ -219,6 +226,9 @@ vtkStandardNewMacro(vtkMRMLSliceLogic);
 //----------------------------------------------------------------------------
 vtkMRMLSliceLogic::vtkMRMLSliceLogic()
 {
+  this->BackgroundLayer = nullptr;
+  this->ForegroundLayer = nullptr;
+  this->LabelLayer = nullptr;
```

Comment:
There appears to be various mishandling of a rebase event? This PR is currently adding back in this code lines which were recently removed in https://github.com/Slicer/Slicer/commit/42a9af984e55ea18d591518632d88b5b2196926c.

Reply:
You're right @jamesobutler, good catch! We haven't actually modified this file in this branch so yes probably a rebase error. I think we can simply revert it and amend the commit. @lassoan do you agree?

Reply:
Yes, of course. Unfortunately, it means that we need to review all the files for such changes.

Reply:
I'm reviewing all the changes now and reverting everything that got in from other commits.